rest/model/test_results.go

```diff
@@ -61,8 +71,9 @@ func (a *APITestResult) Import(i interface{}) error {
 
 // APITTestResultsStats describes basic stats for a group of test results.
 type APITestResultsStats struct {
-	TotalCount  int `json:"total_count"`
-	FailedCount int `json:"failed_count"`
+	TotalCount    int  `json:"total_count"`
+	FailedCount   int  `json:"failed_count"`
+	FilteredCount *int `json:"filtered_count,omitempty"`
```

Comment:
Is this omitempty because it only applies in the pagination case?

Reply:
Yeah, if using the `/stats` route it doesn't make sense to have a `FilteredCount`.